### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 Results window_Screenreader: NVDA does not read loading status of selected “Select top 1000” menu item.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode-mssql#1690]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

**GitHubTags:**#A11y_mssql for VSCode Object Explorer; #A11ySev2; #A11yTCS; #A11yMAS; #DesktopApp; #MAS1.3.1; #Win10; #mssql for VSCode; #NVDA; #JAWS; #FTP; #Benchmark;

**Environment Details:**
Application Name: mssql for VSCode

Screen Reader: NVDA, JAWS 2020

**Repro Steps:** 

1. Open Visual studio code 
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server  button
4. Click on Add Connection Button and insert the valid credentials
5. after adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
6. Select top 1000 window will open navigate to Resluts window
7. Start NVDA and Select top 1000 option.

**Actual:**

NVDA does not read loading status of selected “Select top 1000” menu item.And after opening the ”Select top 1000” window NVDA does not announce proper information.

**Same issue is also observed with throughout the application after performing action for open new code window.**

**Same issue is also observed with JAWS.**

**Expected:**

NVDA should reads loading status that “Select top 1000” window is loading and when “Select top 1000” window will get opened it should announce window information in detail.

**User Impact:**

Screen reader user won’t understand that the proper status of the loading window and information about the “Select top 1000” window.

**Attachment link for Reference:**
[1690_A11y_mssql for VSCode_Select top 1000 Results window_Screenreader.zip](https://github.com/microsoft/vscode-mssql/files/4687652/1690_A11y_mssql.for.VSCode_Select.top.1000.Results.window_Screenreader.zip)

**Comment by hannahqin** [comment]
Updated in conformance doc

**Comment by Roopesh-T** [comment]
Above issue is not repro in env so closing the bug.
![image](https://user-images.githubusercontent.com/86601127/218433630-8909dcb6-3598-4d34-b31e-5e955b851590.png)
![1690_Closed](https://user-images.githubusercontent.com/86601127/218433661-8b01929c-43d1-4b35-b492-90d7f311b295.gif)
**Git Hub Tags**: #Closed;
